Makefile

```diff
@@ -30,11 +30,13 @@ build-rust: build-rust-release
 build-rust-debug:
 	(cd libwasmvm && cargo build)
 	cp libwasmvm/target/debug/libwasmvm.$(DLL_EXT) api
+	cp libwasmvm/bindings.h api
```

Comment:
Logic looks good.

Let's pull this out into a make job like `update-bindings` that allows is to add a comment explaining why the copy is necessary.